### PR TITLE
Update release notes for Katello 4.9.2

### PR DESCRIPTION
**Pull request by wbclark**
Also includes release notes for Katello 4.9.1

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.8/Katello 4.10
* [X] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Comment by ianballou** [comment]
I think this pull request in general only needs to go against the Foreman 3.7 branch. The file where you added forman 3.7.0 and katello 4.9.0 versions should already have those on the 3.7 branch

**Review by ekohl — CHANGES_REQUESTED** [review]
As @ianballou said, this should be against the Foreman branch.

We should enhance the procedure instructions to be explicit about that.

**Comment by wbclark** [comment]
I changed the base branch of the PR but it looks like I'll need to rebase and force push as well

File: guides/doc-Release_Notes/topics/katello-4.9.0.adoc [hunk 1]
```diff
@@ -85,7 +85,6 @@ A full list of changes is available on https://projects.theforeman.org/issues?se
 
 === Repositories
 
-* Can\'t remove GPG and SSL Keys from existing Product using the API - https://projects.theforeman.org/issues/36497[#36497]
```
Comment (wbclark) [inline guides/doc-Release_Notes/topics/katello-4.9.0.adoc:88]: This wasn't included in 4.9.0 but is now included in katello 4.9.2

File: guides/doc-Release_Notes/master.adoc [hunk 2]
```diff
@@ -15,6 +15,9 @@ endif::[]
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.7.0.adoc[leveloffset=+1]
 include::topics/katello-4.9.0.adoc[leveloffset=+1]
+include::topics/katello-4.9.1.adoc[leveloffset=+1]
+include::topics/katello-4.9.2.adoc[leveloffset=+1]
```
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:19]: We order them from oldest to newest. You should also include `ifdef::katello[]` above it and `endif::[]` after. See the 3.6 branch for an example: https://github.com/theforeman/foreman-documentation/blob/3.6/guides/doc-Release_Notes/master.adoc?plain=1

Comment (wbclark) [reply guides/doc-Release_Notes/master.adoc]: OK, I reversed the order and wrapped it in a conditional

**Review by ianballou — APPROVED** [review]
This is looking good to me!

File: guides/doc-Release_Notes/master.adoc [hunk 3]
```diff
@@ -14,7 +14,11 @@ endif::[]
 
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.7.0.adoc[leveloffset=+1]
```
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:16]: I'd list Foreman last. Foreman 3.7.0 was released before Katello 4.9.0.

My reasoning is that you remember I updated to X.Y.Z last, so I read till there. That probably needs to written down somewhere.

File: guides/doc-Release_Notes/topics/katello-contributors.adoc [hunk 4]
```diff
@@ -13,5 +13,12 @@ Partha Aji
 Quinn James
 William Clark
 Markus Bucher
-stbergmann
+Stephan Bergmann
 Trevor Allison
+Maria Agaphontzev
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-contributors.adoc:18]: We keep the list sorted alphabetically.

**Review by ekohl — CHANGES_REQUESTED** [review]
Minor notes, but overall this looks good.

File: guides/doc-Release_Notes/master.adoc (hunk 3 above)
Comment (wbclark) [reply guides/doc-Release_Notes/master.adoc]: I'll look at updating the instructions from the procedure. Thanks!

File: guides/doc-Release_Notes/topics/katello-contributors.adoc (hunk 4 above)
Comment (wbclark) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc]: Done.

**Review by ekohl — COMMENTED** [review]
I'll wait for the preview to render, but from reading the source this looks correct.

File: guides/doc-Release_Notes/topics/katello-4.9.2.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* Custom ACS path help text is missing file:// - https://projects.theforeman.org/issues/36631[#36631]
+
+=== Client/Agent
+
+* tasks Actions::Katello::BulkAgentAction without any sub-plans and stuck in running/pending - https://projects.theforeman.org/issues/36528[#36528]
+* Add UI banners warning users about katello-agent removal - https://projects.theforeman.org/issues/36526[#36526]
+
+=== Content Views
+
+* undefined method \'#single_content_view\' for Katello::Host::ContentFacet::Jail (Katello::Host::ContentFacet) (Safemode::NoMethodError) - https://projects.theforeman.org/issues/36684[#36684]
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-4.9.2.adoc:18]: The script generates incorrect escaping. From https://theforeman-foreman-documentation-preview-pr-2479.surge.sh/3.7/Release_Notes/index-katello.html#_content_views
```
undefined method \'#single_content_view\' for Katello::Host::ContentFacet::Jail (Katello::Host::ContentFacet) (Safemode::NoMethodError) - [#36684](https://projects.theforeman.org/issues/36684)
```

In the past I manually dealt with this. https://github.com/theforeman/foreman-documentation/pull/2427 should properly solve it.

Could you go through the rendered files and check this?

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by wbclark** [comment]
@ekohl I left it as a separate commit for ease of review, with the intent to squash before merging

File: guides/doc-Release_Notes/topics/katello-4.9.0.adoc [hunk 6]
```diff
@@ -71,7 +71,7 @@ A full list of changes is available on https://projects.theforeman.org/issues?se
 * Package upgradable versions are not set correctly based on architecture - https://projects.theforeman.org/issues/36100[#36100]
 * Host cloning is broken - https://projects.theforeman.org/issues/36064[#36064]
 * Setting a Content Source is not persistent - https://projects.theforeman.org/issues/35834[#35834]
-* \'System purpose\' modal doesn\'t reset after switching hosts with switcher - https://projects.theforeman.org/issues/35324[#35324]
+* 'System purpose' modal doesn\'t reset after switching hosts with switcher - https://projects.theforeman.org/issues/35324[#35324]
```
Comment (wbclark) [inline guides/doc-Release_Notes/topics/katello-4.9.0.adoc:74]: `doesn\'t` was rendered as `doesn't` in the page last time, so I left it alone

File: guides/doc-Release_Notes/topics/katello-contributors.adoc [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
-Maria Agaphontzev
-Markus Bucher
-Nagoor Shaik
-Partha Aji
-Quinn James
-Samir Jha
-Sayan Das
-Stephan Bergmann
-Trevor Allison
+Adam Růžička,
```
Comment (wbclark) [inline guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: commas added to match the formatting of foreman-contributors

File: guides/doc-Release_Notes/topics/katello-4.9.0.adoc (hunk 6 above)
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-4.9.0.adoc:74]: Now you understand why my script generated wrong escaping: asciidoc escaping is hard.

File: guides/doc-Release_Notes/topics/katello-contributors.adoc (hunk 7 above)
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: Oh, I like this. It's much easier to maintain than a huge sentence and I didn't think of this. We should standardize on this.

**Review by ekohl — APPROVED** [review]
Looks good. Let me know when it's ready to merge.

**Comment by wbclark** [comment]
Thanks @ekohl , it's squashed and ready for merge

**Comment by ianballou** [comment]
Merging since Katello 4.9.2 is built

**Pull request merged**